gems/smithy/lib/smithy/weld.rb

```diff
@@ -20,5 +20,17 @@ def for?(_model)
     def preprocess(_model)
       # no-op
     end
+
+    # Called when constructing a plan
+    # @return [BuiltInBinding] list of built in bindings for use in endpoint rules.
+    def built_in_bindings
```

Comment:
Theses should probably be "endpoint_built_ins" and "endpoint_functions" with or without binding suffix.